--- 1/Vagrantfile	2017-08-14 18:15:21.000000000 -0300
+++ Vagrantfile	2017-08-14 18:22:49.000000000 -0300
@@ -20,11 +20,13 @@
     end
   end
 
-  config.vm.define "hc1" do |hc|
-    hc.vm.network "private_network", ip: "172.17.6.82"
-    hc.vm.provision "shell", path: "scripts/install-vm", privileged: false
-    hc.vm.provider "virtualbox" do |v|
-      v.memory = 2048
+  (1..2).each do |i|
+    config.vm.define "hc#{i}" do |hc|
+      hc.vm.network "private_network", ip: "172.17.6.8#{i+1}"
+      hc.vm.provision "shell", path: "scripts/install-vm", privileged: false
+      hc.vm.provider "virtualbox" do |v|
+        v.memory = 2048
+      end
     end
   end
 end
